python/cuml/cuml/accel/estimator_proxy.py

```diff
@@ -18,7 +18,8 @@
 from typing import Any
 
 import sklearn
-from sklearn.base import BaseEstimator
+from sklearn.base import BaseEstimator, ClassNamePrefixFeaturesOutMixin
+from sklearn.utils._set_output import _wrap_data_with_container
```

Comment:
Relying on this private method is the cleanest way of handling this. I'm not terribly worried about doing this - the code here hasn't changed measurably since its addition in sklearn 1.2. The only public way to support this is to subclass from `TransformerMixin` and rely on its `__init_subclass__` method to do the right thing, but this doesn't play nicely with how the proxy works and would end up relying on other parts of `_SetOutputMixin` not changing their implementation in a more fragile way.

I'm happy with this choice, just flagging it to justify it.